internal/app/machined/pkg/controllers/network/node_address_test.go

```diff
@@ -243,32 +248,47 @@ func (suite *NodeAddressSuite) TestFilters() {
 						case network.NodeAddressCurrentID:
 							if !reflect.DeepEqual(
 								addrs,
-								ipList("1.2.3.4/32 10.0.0.1/8 25.3.7.9/32 2001:470:6d:30e:4a62:b3ba:180b:b5b8/64"),
+								ipList("1.2.3.4/32 10.0.0.1/8 25.3.7.9/32 2001:470:6d:30e:4a62:b3ba:180b:b5b8/64 fdae:41e4:649b:9303:7886:731d:1ce9:4d4/128"),
```

Comment:
assert.Equal doesn't work here?

Reply:
it's an "old" test, we can't use assert, as it's not compatible with `retry`, as we want to return an error instead of recording a test failure